### PR TITLE
Redesign detail infopraktikal

### PR DESCRIPTION
**Pull request by gaganugraha33**
- Redesain detail info praktikal
- delete code yang tidak digunakan

[[S14] Redesain Detail Info Praktikal](https://trello.com/c/d2WEP4lm)

### CODE REVIEW

**Comment by gaganugraha33** [comment]
review gan @anandarh @jaluowen @aldiiii

File: lib/components/CollapsingAppbar.dart [hunk 1]
```diff
@@ -47,6 +47,7 @@ class _CollapsingAppbarState extends State<CollapsingAppbar> {
     /// for set default value for height appbar & bottom Appbar
     heightAppbar = widget.heightAppbar ?? 150;
     isBottomAppbar = widget.isBottomAppbar ?? true;
+    print('tingginya berapa? '+heightAppbar.toString());
```
Comment (aldiiii) [inline lib/components/CollapsingAppbar.dart:50]: remove print

Comment (gaganugraha33) [reply lib/components/CollapsingAppbar.dart:50]: eh iya wkwkw

Comment (gaganugraha33) [reply lib/components/CollapsingAppbar.dart]: udah di hapus mang @aldiiii

**Review by jaluowen — APPROVED** [review]

**Review by anandarh — APPROVED** [review]

**Pull request merged**